docs/SyslogApi.md

```diff
@@ -0,0 +1,394 @@
+
+### Return type
+
+void (empty response body)
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+ - **Content-Type**: Not defined
+ - **Accept**: Not defined
+
+[[Back to top]](#) [[Back to API list]](index.md#endpoint-properties) [[Back to Model list]](index.md#documentation-for-models) [[Back to Overview]](index.md)
+
+# **list_syslog_servers**
+> SyslogServerResponse list_syslog_servers(filter=filter, ids=ids, limit=limit, names=names, sort=sort, start=start, token=token)
+
+
+
+Return a list of configured syslog servers.
+
+### Example 
+```python
+from purity_fb import PurityFb, rest
+
+fb = PurityFb("10.255.9.28", version=__version__)  # assume the array IP is 10.255.9.28
+fb.disable_verify_ssl()
+try:
+    res = fb.login(API_TOKEN) # login to the array with your API_TOKEN
+except rest.ApiException as e:
+    print("Exception when logging in to the array: %s\n" % e)
+if res:
+    try:
+        # List all configured syslog servers
+        res = get_client().syslog.list_syslog_servers()
+        print(res)
+
+        # List first two syslog servers beginning with 'syslog_fb'. Use default sorting.
+        res = fb.syslog.list_syslog_servers(limit=2, names=["syslog_fb*"])
+        print(res)
+
+        # List the first syslog server when sorting by name.
+        res = fb.syslog.list_syslog_servers(limit=1, sort="name")
+        print(res)
+
+        # List all syslog servers using TCP connections
+        res = fb.syslog.list_syslog_servers(filter='uri=\'tcp*\'')
+        print(res)
+    except rest.ApiException as e:
+        print("Exception when listing syslog servers: %s\n" % e)
+```
+
+### Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **filter** | **str**| The filter to be used for query. | [optional] 
+ **ids** | **list[str]**| A comma-separated list of resource IDs. This cannot be provided together with the name or names query parameters. | [optional] 
+ **limit** | **int**| limit, should be &gt;&#x3D; 0 | [optional] 
+ **names** | **list[str]**| Performs the operation on the unique name specified. Enter multiple names in comma-separated format. For example, &#x60;name01,name02&#x60;. | [optional] 
+ **sort** | **str**| The way to order the results. | [optional] 
+ **start** | **int**| start | [optional] 
+ **token** | **str**| token | [optional] 
+
+### Return type
+
+[**SyslogServerResponse**](SyslogServerResponse.md)
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+ - **Content-Type**: Not defined
+ - **Accept**: Not defined
+
+[[Back to top]](#) [[Back to API list]](index.md#endpoint-properties) [[Back to Model list]](index.md#documentation-for-models) [[Back to Overview]](index.md)
+
+# **list_syslog_settings**
+> SyslogServerSettingResponse list_syslog_settings(filter=filter, ids=ids, limit=limit, names=names, sort=sort, start=start, token=token)
```

Comment:
I don't think they can be hidden here without outright removing them as parameters (which I don't think we want to do). I don't think this is _too_ bad, since we show these same optional parameters for all of our `list` operations, regardless of usefulness.